### ISSUE TITLE
Validate - Highlight Line

### ISSUE THREAD

**Issue by SonGokussj4** [fernandreu/office-ribbonx-editor#47]
Hello, I really like your refresh of the Editor.

I have a suggestion. When I press `Validate` and it will just show me MsgBox with details, it would be quite nice to have then highlighted the compromised line. 

Any chance this can be made or is a good idea? :-)

**Comment by fernandreu** [comment]
Hi SonGokussj4,

Thanks for the suggestion! It is definitely feasible with some minor changes, and I have implemented a working version for this in 52b99a9.

However, I also just realised it can be a bit annoying if you have multiple XML validation errors at once, as each one will have its own message box that will be shown sequentially. To avoid this, I will do more in-depth changes to the FindAllResultsPanel (#43) so that the XML validation errors can also be shown there. Something like:

```
Line 8, Position 3: Unknown XML tag 'abc'
Line 12, Position 6: Another XML validation error
```

When clicking one of those results, the corresponding line in the editor would get selected.
